### PR TITLE
Feature Branch to master

### PR DESCRIPTION
**Pull request by DickensJuma**
### Design - Tear Down Project
[https://raw.githack.com/DickensJuma/Design-TearDown/feature/index.html](url)
- Created a heatmap of the smashing magazine.
the project consist of :
- navigation bar 
- section having articles
- positioned using grid and flex

### CODE REVIEW

**Comment by lekegitrepo** [comment]
### Hi @DickensJuma and @glowreeyah 👏 👏 
### _Well done, you have done a great job on this project but there are some requirements you need to implement before this project can be approved._

**_1. Files not related to the project should not be committed and added to .gitignore_** (e.g. `.vscode`) ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/general/gitignore.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/66243643/))

**_2. Elements should be aligned with_** [float](https://css-tricks.com/all-about-floats/) and [flex](https://css-tricks.com/snippets/css/a-guide-to-flexbox/) or [grid](https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_Grid_Layout) ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/html_css/design_teardown/align_requirement.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/59540025/))

**_I think float is missing in the style rules_**

**_3. The footer should contain centered links_** (content is not important) ([link to page](https://www.smashingmagazine.com/))

**_you can wrap the text in the li of topic class with a link tag._**

**_Kindly implement these changes and submit for review_**

### Review by: _Temitayo Adeleke_

**Review by lekegitrepo — CHANGES_REQUESTED** [review]
### Status: Require changes

File: style.css [hunk 1]
```diff
@@ -0,0 +1,566 @@
...
+  background-color: #f6f3f2;
+}
+
+.photo-name {
+  display: flex;
+  margin: 200px 100px;
+}
+
+.photo-name p {
+  font-size: 20px !important;
```
Comment (uryelah) [inline style.css]: - [ ] Don't use !important in your style. [Read more about specificity in CSS here](https://developer.mozilla.org/en-US/docs/Web/CSS/Specificity) for alternatives if you are having issues applying this rule.

File: style.css [hunk 2]
```diff
@@ -0,0 +1,566 @@
...
+  position: relative;
+}
+
+.navbar {
+  text-align: center;
+}
+
+.navbar > a {
+  margin: 0 auto;
+  color: rgb(255, 248, 248);
```
Comment (uryelah) [inline style.css]: - [ ] [ OPTIONAL] this project should be made all in tones of grey.
The way you make a shade of grey in rgb is by using **the same number for the three arguments**, the way you can make a grey using hex colors is by using the same string three times, ex: #a1a1a1.
A lot of the 'greys' you are using still have color in them. Try to apply the rules mentioned above to all the places you set color and background color. 
The reason for that is that the heatmap is about showing the visual weight of each visual element of the page and when you have color near black and whites and greys, even if it's very muted, it calls a lot of visual weight to itself, thus defeating the objective of the heat map.

[This is a good reference on shades of grey,](https://www.cssportal.com/html-colors/shades-of-gray.php) notice the number patterns.

**Review by uryelah — CHANGES_REQUESTED** [review]
Hello **Glory David David-Akubo** & **Juma Juma**, I'm Sarah, your reviewer for this PR 👩‍🚀  

**First of all, great job with the positioning and using transforms for the elements 🌟**

Your project is almost good to go, you are just missing one requirement. You got this! 💪

---------    

## Required Changes (check the comments)

- [ ] Remember to change the default style of the links. They should be grey of black, not blue.
Screenshot_2020-03-25 Smash Magazine

**Comment by uryelah** [comment]
**Kindly fix the REQUIRED issues and send another code review request.**

Cheers and Happy coding!👏👏👏

Code review by **Sarah Uryelah Silva Chamorro**

(Feel free to contact me on slack by the handle of  **sarah** if you have any additional question)

**Comment by uryelah** [comment]
**Kindly fix the REQUIRED issues and send another code review request.**

Cheers and Happy coding!👏👏👏

Code review by **Sarah Uryelah Silva Chamorro**

(Feel free to contact me on slack by the handle of  **sarah** if you have any additional question)

**Comment by mikenath223** [comment]
### Pull request currently under review by Michgolden Ukeje (TSE).
Kindly exercise patience 🐱

**Review by mikenath223 — APPROVED** [review]
### CONGRATULATIONS 👏👏👏

**Great Job @DickensJuma  and @glowreeyah**  🎉

I see you've worked hard with the reviewer on following the needed requirements.

Therefore I am approving this project.

**STATUS: APPROVED**

Although there remains one aspect that needs to be fixed. But I trust you will do that before merging.

Kindly do this before merging: 👇

- [ ] The newsletter subscription section that comes just before the footer section is supposed to be full width and fill the entire page section it occupies, it shouldn't have an element beside it like yours👇 [Screenshot]:ice_screenshot_20200325-193240

It's supposed to look this way👇[Screenshot]:ice_screenshot_20200325-191943

Reach out to me on slack(michgolden) if you need extra help on this.

Good luck with the next project. 👍

**HAPPY CODING👨‍💻**

**Comment by glowreeyah** [comment]
> ### CONGRATULATIONS 👏👏👏
> **Great Job @DickensJuma and @glowreeyah** 🎉
> 
> I see you've worked hard with the reviewer on following the needed requirements.
> 
> Therefore I am approving this project.
> 
> **STATUS: APPROVED**
> 
> Although there remains one aspect that needs to be fixed. But I trust you will do that before merging.
> 
> Kindly do this before merging: 👇
> 
> * [ ]  The newsletter subscription section that comes just before the footer section is supposed to be full width and fill the entire page section it occupies, it shouldn't have an element beside it like yours👇 [Screenshot]:ice_screenshot_20200325-193240
> 
> It's supposed to look this way👇[Screenshot]:ice_screenshot_20200325-191943
> 
> Reach out to me on slack(michgolden) if you need extra help on this.
> 
> Good luck with the next project. 👍
> 
> **HAPPY CODING👨‍💻**

Hi Michgolden,
Thank you for the approval! However, I think what your browser shows is a bit different from what we have here. 
screenshot-www smashingmagazine com-2020 03 25-22_54_03

**Pull request merged**